LICENSE-3rdparty.yml

```diff
@@ -32481,6 +32481,33 @@ third_party_libraries:
+- package_name: urlencoding
```

Comment:
/question

what is our policy on using the dependencies? 

It looks like this crate is managed by some independent developer which seems risky.

Reply:
There are policies against libraries with certain licenses, but there is no policy against using libraries from independent developers. We use many libraries from independent open source maintainers. `urlencoding` is fairly popular.